examples/ibm-openshift-job/README.MD

```diff
@@ -0,0 +1,25 @@
+# terraform_cluster_job
```

Comment:
Can you provide  a brief heading what exactly this configuration..Running a job in kubernetes cluster ....